### PR TITLE
error messages in HTML pages updates

### PR DESCRIPTION
**Pull request by alisonhofer**
I am meeting with David, Jake, and Mark tomorrow to learn more about LaTeXML and the converter we use. I plan on obtaining more information about TikZ so I can add more to that section in FAQ. l

I am also seeking suggestions for what else we might want to say at the top in the introduction.

### CODE REVIEW

File: source/about/accessibile_html_error_messages.md [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
+The errors that you have seen when viewing papers in HTML after submission or on arXiv.org are caused when the LaTeXML converter is unable to translate certain TeX and LaTeX software packages. TeX and LaTeX, pronounced ‘tek’ and ‘LAH-tek’, are open source software packages used to accurately display complex mathematical formulae, symbols and tables in research papers. Due to the open source nature of TeX and LaTeX and its popularity, there are many packages available to choose from. This makes supporting all of them very difficult. NIST is routinely assessing and adding LaTeX packages in an effort to provide greater versatility to the LaTeXML converter.
```
Comment (SBBCornell) [inline source/about/accessibile_html_error_messages.md:6]:
```suggestion
The errors that you have seen when viewing papers in HTML after submission or on arXiv.org are caused when the LaTeXML converter is unable to translate certain TeX and LaTeX software packages. TeX and LaTeX, pronounced ‘tek’ and ‘LAH-tek’, are open source software packages used to accurately display complex mathematical formulae, symbols and tables in research papers. Due to the open source nature of TeX and LaTeX and its popularity, there are many packages available to paper authors, and new ones being added often. This makes supporting all of them very difficult. NIST is routinely assessing and adding LaTeX packages in an effort to provide greater versatility to the LaTeXML converter.
```

File: source/about/accessibile_html_error_messages.md [hunk 2]
```diff
@@ -1,58 +1,58 @@
...
 
-Please note: These errors are only the result of our  inability to properly translate and display certain TeX and LaTex packages as HTML. These errors are not an indication of a larger system failure.
 
-## HTML errors readers may encounter
+## HTML errors you may see:
 
 When browsing and reading HTML papers on arXiv.org, you may see formatting errors in red like below: 
 
-![HTML LaTeX processing error for online id](images/reader-error-01.png)
+![HTML LaTeX processing error for online id](images/reader-error-01.png){.mkd-img-border}
```
Comment (SBBCornell) [inline source/about/accessibile_html_error_messages.md:13]: This an the next two images need alt text

File: source/about/accessibile_html_error_messages.md [hunk 3]
```diff
@@ -1,58 +1,58 @@
...
+![HTML LaTeX processing error for system name and revision](images/reader-error-03.png){.mkd-img-border}
 
 
-
-## HTML errors authors and submitters may encounter
-
-After you have submitted your paper and it has been processed, you will receive a link to preview the HTML version of your paper. You may see an error message alerting you that the TeX/LaTeX packages used to format your paper are not fully supported by our conversion process. Therefore your paper may display red TeX/LaTeX code and other formatting irregularities. 
-
 Please see an example of a HTML processing error below:
 ![html processing error](images/author_submssn_error.png){.mkd-img-full alt='Error message in a red box stating, This paper uses packages, listed below, that do not yet convert to HTML. These issues are known and are being worked on. View the list of unsupported packages.'}
```
Comment (SBBCornell) [inline source/about/accessibile_html_error_messages.md:21]: this is great alt text!

File: source/about/accessibile_html_error_messages.md [hunk 4]
```diff
@@ -1,58 +1,58 @@
...
 LaTeX is a TeX macro package developed in the early 1980s by [Leslie Lamport](https://en.wikipedia.org/wiki/LaTeX#cite_note-Lamport1986-4). It provides the ability to structure a document to meet different layout requirements such as section headers, footnotes, graphics and bibliographies.
 
-### Why are TeX and LaTex  so popular? 
-They are beloved by academic communities for their ability to easily display complex mathematical formulae, symbols, charts and graphs that common word processing programs have difficulty with. 
+### What is TeXML? 
+TeXML is an open source process that describes the markup elements of TeX using XML syntax. This allows XML data to be presented in PDF.
 
-TeX and LaTex are open source software programs and are supported, maintained and iterated upon by the communities that use them. As a result, there are many packages available offering a myriad of features to format research papers. 
+### What is LaTeXML?
+LaTeXML is an open source software package that converts LaTeX documents to XML, HTML, EPUB, JATS and TEI.
```
Comment (SBBCornell) [inline source/about/accessibile_html_error_messages.md:35]: Can we add here that it is authored by the NIST team of Bruce Miller and Deyan Ginev?

File: source/about/accessibile_html_error_messages.md [hunk 5]
```diff
@@ -1,58 +1,58 @@
...
 TeX is a text formatting and markup language first released in late 1970s by [D. Knuth](https://en.wikipedia.org/wiki/TeX#Hyphenation_and_justification). It was created to properly configure and visualize mathematical formulae, tables and symbols in printed documents such as journals and books. 
 
-### What is LaTex?
+### What is LaTeX?
 LaTeX is a TeX macro package developed in the early 1980s by [Leslie Lamport](https://en.wikipedia.org/wiki/LaTeX#cite_note-Lamport1986-4). It provides the ability to structure a document to meet different layout requirements such as section headers, footnotes, graphics and bibliographies.
 
-### Why are TeX and LaTex  so popular? 
-They are beloved by academic communities for their ability to easily display complex mathematical formulae, symbols, charts and graphs that common word processing programs have difficulty with. 
+### What is TeXML? 
+TeXML is an open source process that describes the markup elements of TeX using XML syntax. This allows XML data to be presented in PDF.
```
Comment (SBBCornell) [inline source/about/accessibile_html_error_messages.md:32]: do we use this tool? I am not aware of arXiv calling this tool, only latexml. Or does LaTeXML use this tool?

File: source/about/accessibile_html_error_messages.md [hunk 6]
```diff
@@ -1,58 +1,58 @@
...
+
+TeX and LaTeX are open source software programs  that  are platform independent which allows for the ability to view documents regardless of the operating system or software set-up making collaboration simple. These programs are maintained and iterated upon by the communities that use them, and as a result, there are many packages available offering a myriad of features to format research papers. 
 
-### Why are TeX and LaTex so difficult to support?
-Due to the robust open source community supporting and developing TeX and LaTex packages, there are an enormous number of packages in use. We are continuously reviewing which features and packages are most popular in an effort to provide error free, accessible HTML papers.
+### Why are TeX and LaTeX so difficult to support?
+Due to the robust open source community supporting and developing TeX and LaTeX packages, there are many packages in use. NIST is continuously reviewing which features and packages are most popular to add to the LaTeXML converter in an effort to provide error free, accessible HTML papers.
```
Comment (SBBCornell) [inline source/about/accessibile_html_error_messages.md:43]:
```suggestion
Due to the robust open source community supporting and developing TeX and LaTeX packages, there are many packages in use. NIST is continuously reviewing which features and packages are most popular to add to the LaTeXML converter in an effort to provide high fidelity, accessible conversions to HTML.
```

File: source/about/accessibile_html_error_messages.md [hunk 7]
```diff
@@ -1,58 +1,58 @@
...
 
-### Which TeX/LaTeX packages do you or do not support?
+### Which TeX/LaTeX packages are supported?
 - Please see our list of supported packages
 - Please see our list of [unsupported packages](https://github.com/brucemiller/LaTeXML/wiki/Porting-LaTeX-packages-for-LaTeXML)
 
-## Do you support TikZ?
-See What are 'partially' supported packages?
+### Is TikZ supported?
+Sort of. See What are 'partially' supported packages?
```
Comment (SBBCornell) [inline source/about/accessibile_html_error_messages.md:50]:
```suggestion
Partially. See the next section below.
```

File: source/about/accessibile_html_error_messages.md [hunk 8]
```diff
@@ -1,58 +1,58 @@
...
+
+### What are ‘partially’ supported packages? 
 
-## What are ‘partially’ supported packages? 
+A partially supported package is a package that has select features NIST has chosen to support in LaTeXML. For example, TikZ offers the ability to … (need to add list of features) and LaTeXML supports …. (add list of features supported) However, TikZ also provides the ability to… which LaTeXML does not support.
```
Comment (SBBCornell) [inline source/about/accessibile_html_error_messages.md:54]:
```suggestion
A partially supported package has select features NIST is supporting now in LaTeXML, but other features that are not yet supported. For example, TikZ offers the ability to … (need to add list of features) and LaTeXML supports …. (add list of features supported) However, TikZ also provides the ability to… which LaTeXML does not support.
```

File: source/about/accessibile_html_error_messages.md [hunk 9]
```diff
@@ -1,58 +1,58 @@
...
 
-A partially supported package is a package that has select features we support in research papers.
+### What is NIST?
```
Comment (SBBCornell) [inline source/about/accessibile_html_error_messages.md:56]: Do we need this section? There is a link to NIST in an earlier section. That seems like enough.

Comment (SBBCornell) [reply source/about/accessibile_html_error_messages.md:56]: One more recommendation: a section for authors about how they can help reduce HTML errors and make papers more accessible by using packages that LaTeXML fully supports.

**Comment by SBBCornell** [comment]
One more recommendation: a section for authors about how they can help reduce HTML errors and make papers more accessible by using packages that LaTeXML fully supports.

File: source/about/accessibile_html_error_messages.md (hunk 5 above)
Comment (alisonhofer) [reply source/about/accessibile_html_error_messages.md]: I have updated the wording here and you will see it when I update this PR.

File: source/about/accessibile_html_error_messages.md (hunk 2 above)
Comment (alisonhofer) [reply source/about/accessibile_html_error_messages.md:13]: I have added alt text.

File: source/about/accessibile_html_error_messages.md [hunk 10]
```diff
@@ -1,58 +1,61 @@
 # Error messages in HTML papers
 
-The process of converting our papers to HTML is a work in progress and as we improve the process you will see fewer errors and formatting issues. 
+HTML papers on arXiv.org will sometimes display errors. To produce accessible HTML papers we use a LaTeXML converter created and maintained by National Library of Standards and Technology (NIST). As we work to improve accessibility we share with you the causes of these errors and what authors can do to help minimize them.
```
Comment (SBBCornell) [inline source/about/accessibile_html_error_messages.md:3]:
```suggestion
HTML papers on arXiv.org are a work in progress and will sometimes display errors. As we work to improve accessibility we share with you the causes of these errors and what authors can do to help minimize them.
```

File: source/about/accessibile_html_error_messages.md [hunk 11]
```diff
@@ -1,58 +1,61 @@
...
+The errors that you have seen when viewing papers in HTML after submission or on arXiv.org are caused when the LaTeXML converter is unable to translate certain TeX and LaTeX software packages. TeX and LaTeX, pronounced ‘tek’ and ‘LAH-tek’, are open source software packages used to accurately display complex mathematical formulae, symbols and tables in research papers. Due to the open source nature of TeX and LaTeX and its popularity, there are many packages available to choose from. This makes supporting all of them very difficult. NIST is routinely assessing and adding LaTeX packages in an effort to provide greater versatility to the LaTeXML converter.
```
Comment (SBBCornell) [inline source/about/accessibile_html_error_messages.md:6]:
```suggestion
To produce accessible HTML papers we use a converter created and maintained by the LaTeXML team at the National Library of Standards and Technology (NIST). 

The errors that you have seen when viewing papers in HTML after submission or on arXiv.org are caused when the LaTeXML converter is unable to translate certain TeX and LaTeX software packages. TeX and LaTeX, pronounced ‘tek’ and ‘LAH-tek’, are open source software packages used to accurately display complex mathematical formulae, symbols and tables in research papers. Due to the open source nature of TeX and LaTeX and its popularity, there are many packages available to choose from. This makes supporting all of them very difficult. NIST is routinely assessing and adding LaTeX packages in an effort to provide greater versatility to the LaTeXML converter.
```

File: source/about/accessibile_html_error_messages.md [hunk 12]
```diff
@@ -1,58 +1,61 @@
...
 
-### What is Tex?
-TeX is a text formatting and markup language first released in late 1970s by [D. Knuth](https://en.wikipedia.org/wiki/TeX#Hyphenation_and_justification). It was created to properly configure and visualize mathematical formulae, tables and symbols in printed documents such as journals and books. 
+### What is LaTeXML?
 
-### What is LaTex?
-LaTeX is a TeX macro package developed in the early 1980s by [Leslie Lamport](https://en.wikipedia.org/wiki/LaTeX#cite_note-Lamport1986-4). It provides the ability to structure a document to meet different layout requirements such as section headers, footnotes, graphics and bibliographies.
+LaTeXML was developed at the [Digital Library of Mathematical Functions at NIST](https://math.nist.gov/) (National Institute of Standards and Technology) and is authored by Bruce Miller and Deyan Ginev. It's primary output is XML and a postprocessor can convert these XML documents into HTML with mathematical forumulas as images or XHTML, HTML5, EPUB with formulas as MathML.
```
Comment (SBBCornell) [inline source/about/accessibile_html_error_messages.md:38]:
```suggestion
LaTeXML was developed at the [Digital Library of Mathematical Functions at NIST](https://math.nist.gov/) (National Institute of Standards and Technology) and is authored by Bruce Miller and Deyan Ginev. It's primary output is XML and a postprocessor can convert these XML documents into HTML with mathematical formulas as MathML. 
```

File: source/about/accessibile_html_error_messages.md [hunk 13]
```diff
@@ -1,58 +1,61 @@
...
-## Do you support TikZ?
-See What are 'partially' supported packages?
+### Is TikZ supported?
+Partially, please see 'What are partially supported packages?' below.
 
-## What are ‘partially’ supported packages? 
+### What are ‘partially’ supported packages? 
 
-A partially supported package is a package that has select features we support in research papers.
+A partially supported package is a package that has select features NIST has chosen to support in LaTeXML. For example, TikZ offers the ability to … (need to add list of features) and LaTeXML supports …. (add list of features supported) However, TikZ also provides the ability to… which LaTeXML does not support.
```
Comment (SBBCornell) [inline source/about/accessibile_html_error_messages.md:60]:
```suggestion
A partially supported package has select features NIST has chosen to support in LaTeXML. For example, TikZ offers the ability to … (need to add list of features) and LaTeXML supports …. (add list of features supported) However, TikZ also provides the ability to… which LaTeXML does not support.
```

File: source/about/accessibile_html_error_messages.md [hunk 14]
```diff
@@ -1,58 +1,61 @@
...
 
-### Why are TeX and LaTex so difficult to support?
-Due to the robust open source community supporting and developing TeX and LaTex packages, there are an enormous number of packages in use. We are continuously reviewing which features and packages are most popular in an effort to provide error free, accessible HTML papers.
+### Why are TeX and LaTeX so difficult to support?
+In progress...
 
-### Which TeX/LaTeX packages do you or do not support?
-- Please see our list of supported packages
+### Which TeX/LaTeX packages are supported?
+- Please see our list of [supported packages](https://corpora.mathweb.org/corpus/arxmliv/tex_to_html/info/loaded_file)
```
Comment (SBBCornell) [inline source/about/accessibile_html_error_messages.md:52]:
```suggestion
- Please see this list of [supported packages](https://corpora.mathweb.org/corpus/arxmliv/tex_to_html/info/loaded_file)
```

File: source/about/accessibile_html_error_messages.md [hunk 15]
```diff
@@ -1,58 +1,61 @@
...
 
-![HTML LaTeX processing error for system name and revision](images/reader-error-03.png)
+![HTML LaTeX processing error for tik Z library](images/reader-error-02.png){.mkd-img-border alt='Tik Z errors are displayed in red markup. Errors are: foward slash use tik z library, foward slash p.g.f. declare decorations simple line initial'}
 
+![HTML LaTeX processing error for system name and revision](images/reader-error-03.png){.mkd-img-border alt='System name errors are displayed in red markup. Errors are: forward slash system name, foward slash revision'
+}
 
+## Steps authors can take to minimize HTML errors
+Authors and proxies who submit papers to arXiv can help minimize HTML errors by formatting submissions using one of the LateX packages fully supported by LaTeXML. [Please view the list of fully supported LaTeX packages](https://corpora.mathweb.org/corpus/arxmliv/tex_to_html/info/loaded_file).
```
Comment (SBBCornell) [inline source/about/accessibile_html_error_messages.md:23]:
```suggestion
Authors who submit papers to arXiv can help minimize HTML errors by using LateX packages that are fully supported by LaTeXML. [Please view the list of fully supported LaTeX packages](https://corpora.mathweb.org/corpus/arxmliv/tex_to_html/info/loaded_file).
```

**Review by SBBCornell — COMMENTED** [review]
I added a few more comments. Thanks Alison!

File: source/about/accessibile_html_error_messages.md [hunk 16]
```diff
@@ -1,58 +1,63 @@
...
+They are beloved by academic communities for their ability to easily display complex mathematical formulae, symbols, charts and graphs that common word processing programs have difficulty rendering.
 
-TeX and LaTex packages are platform independent which allows for the ability to view documents regardless of the operating system or software set up making collaboration simple.
+TeX and LaTeX are open source software programs  that  are platform independent which allows for the ability to view documents regardless of the operating system or software set-up making collaboration simple. These programs are maintained and iterated upon by the communities that use them, and as a result, there are many packages available offering a myriad of features to format research papers. 
 
-### Why are TeX and LaTex so difficult to support?
-Due to the robust open source community supporting and developing TeX and LaTex packages, there are an enormous number of packages in use. We are continuously reviewing which features and packages are most popular in an effort to provide error free, accessible HTML papers.
+### Why are TeX and LaTeX so difficult to support?
```
Comment (cbf66) [inline source/about/accessibile_html_error_messages.md:50]: Why are we asking a question that suggests arXiv's whole strategy on this was a bad idea?

**Review by cbf66 — COMMENTED** [review]
Some adjustments. Some things here seem to have gotten off track.

File: source/about/accessibile_html_error_messages.md (hunk 16 above)
Comment (alisonhofer) [reply source/about/accessibile_html_error_messages.md:50]: At some point I recalled a discussion about why is it so challenging to support TeX and LaTeX. This was by no means meant to imply this strategy was a bad idea. I thought it more as managing expectations and indicating not all packages will be supported.

File: source/about/accessibile_html_error_messages.md [hunk 17]
```diff
@@ -1,58 +1,63 @@
...
+To produce accessible HTML papers we use a converter created and maintained by the LaTeXML team at the National Library of Standards and Technology (NIST). 
 
-Please note: These errors are only the result of our  inability to properly translate and display certain TeX and LaTex packages as HTML. These errors are not an indication of a larger system failure.
+The errors that you have seen when viewing papers in HTML after submission or on arXiv.org are caused when the LaTeXML converter is unable to translate certain TeX and LaTeX software packages. TeX and LaTeX, pronounced ‘tek’ and ‘LAH-tek’, are open source software packages used to accurately display complex mathematical formulae, symbols and tables in research papers. Due to the open source nature of TeX and LaTeX and its popularity, there are many packages available to choose from. This makes supporting all of them very difficult. NIST is routinely assessing and adding LaTeX packages in an effort to provide greater versatility to the LaTeXML converter.
```
[the remark this answers is not in the document]
Comment (alisonhofer) [reply source/about/accessibile_html_error_messages.md:8]: I can update the pronunciation and follow what is suggested on this page: [introduction to LaTeX](https://www.latex-project.org/about/#:~:text=LaTeX%2C%20which%20is%20pronounced%20%C2%ABLah,is%20not%20a%20word%20processor!) or leave out pronunciation all together.

**Pull request merged**